app/controllers/chef_ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ChefIngredientsController < ApplicationController
+  def show
+    @chef = Chef.find(params[:id])
```

Comment:
I agree with your controller organization here! Nice work. Keep in mind the @ingredients instance var isn't strictly necessary as you can call @chef.ingredients_used in the view too